### ISSUE TITLE
Better color coding in Visual Studio Code extension

### ISSUE THREAD

**Issue by WeslomPo** [YarnSpinnerTool/YarnSpinner#253]
Hi!

Recently I started writing some quests for my game, and I struggle with Visual Studio Code editor. There are to small difference between keywords so I did not see difference some times. Maybe you can made it more colorfull?
![image](https://user-images.githubusercontent.com/310008/84562321-aed8cc80-ad64-11ea-97c3-04211bcf08c8.png)

To something like this ? (colored by hand)

![image](https://user-images.githubusercontent.com/310008/84562484-e7c57100-ad65-11ea-9867-24f4f193d846.png)

**Comment by WeslomPo** [comment]
Can you share an extension source code? I want to improve it by myself, but making it from scratch - is a bit painful.

**Comment by desplesda** [comment]
Yes! The source code to the extension is available at this repo: https://github.com/YarnSpinnerTool/VSCodeExtension
